client/src/views/Login.vue

```diff
@@ -93,7 +93,7 @@ export default {
       // let email = this.email
       // let password = this.password
       this.$store
-        .dispatch("login", { data: this.loginDetails })
+        .dispatch("login", this.loginDetails )
```

Comment:
I wasn't quite sure on this one either, as I'm not sure how you were getting around it. But I couldn't quickly find a way to get Passport to look within the data field for the user credentials.